src/cr/cube/slices.py

```diff
@@ -167,7 +161,7 @@ def name(self):
     @lazyproperty
     def ndim(self):
         """int count of dimensions for this slice, unconditionally 2."""
-        return 2
+        return self._matrix.ndim
```

Comment:
Why is this the right thing to do? What value other than 2 can it have and why? It breaks a fundamental invariant about a Slice object if this can vary, at least on the public interface.